packages/fluentui/accessibility/src/behaviors/Card/cardBehavior.ts

```diff
@@ -5,12 +5,14 @@ import { Accessibility } from '../../types';
  * @specification
  * Adds role='group'.
  * Adds attribute 'aria-disabled=true' based on the property 'disabled'.
+ * Adds attribute 'aria-selected=true' based on the property 'selected'.
  */
 const cardBehavior: Accessibility<CardBehaviorProps> = props => ({
   attributes: {
     root: {
       role: 'group',
       'aria-disabled': props.disabled,
+      'aria-selected': props.selected,
```

Comment:
`aria-selected` is not valid for `group`: https://www.w3.org/WAI/PF/aria/roles#group - if we tested that screen readers accept it, we need to document it.